src/widget/wbattery.cpp

```diff
@@ -119,20 +124,20 @@ void WBattery::update() {
                                               m_dischargingPixmaps.size())];
             }
             if (minutesLeft == Battery::TIME_UNKNOWN) {
-                setBaseTooltip(tr("Time left unknown."));
+                appendBaseTooltip(tr("Time left unknown."));
```

Comment:
The same here. Can we just remove this? What do you think?

Reply:
done